src/ScreenshotManager.vala

```diff
@@ -162,6 +178,44 @@ namespace Gala
 			selection_area.get_selection_rectangle (out x, out y, out width, out height);
 		}
 
+		private void unconceal_text ()
+		{
+			if (!text_concealed) {
+				return;
+			}
+
+			var desktop_settings = new Settings ("org.gnome.desktop.interface");
```

Comment:
Do not re-create settings everytime this method is called, make them a static member inside the `ScreenshotManager` class and initialize in `static construct`.

Reply:
Considering `ScreenshotManager` already a singleton, I think a non-static field and `construct` makes more sense.